myfirstproject/index.html

```diff
@@ -0,0 +1,93 @@
+    </article>
+    <article>
+    <h2>Why Haskell?</h2>
+    </article>
+    <article>
+       <p> Haskell is closely aligned to conceptual mathematics and number theory, the inspiration for Gattegno Mathematics. It has four attributes that make it particularly appropriate for mathematics education:</p>
+    </article>
+    <ul>
+        <li>Its syntax is very close to traditional mathematical notation (but beware that it can be too terse).</li>
+        <li>Variables are statically typed, which means that programmers are protected by the compiler from passing invalid information as an argument (input) or output value.</li>
+        <li>The Haskell interpreter evaluates functions ‘lazily’, only when required. This makes it possible to program infinite data structures in finite memory.</li>
+        <li>Information is passed from the arguments to functions by pattern matching. This is an intuitive way for programmers to envisage and reason about the behaviour of their programs.</li>
+    </ul>
+    <figure>
+        <img src="../myfirstproject/img/1.jpg" alt="steps order">
+        <figcaption>The ten Cuisenaire rods arranged in order as a staircase with step size White.</figcaption>
+    </figure>
+    <figure>
+        <img src="../myfirstproject/img/page1_1.jpg" alt="math">
+        <figcaption>Equivalent expressions using Gattegno’s rod codes to represent the relationship between a rod and equivalent trains of one colour</figcaption>
+    </figure>
+    <article>
+        <h2>Starting with a staircase</h2>
+    </article>
+    <article>
+        <h2>To start, we went back to a familiar pattern that Gattegno introduces in the textbooks, called a staircase. This is built from one rod of each colour arranged in size order (figure 1). The children were able to physically build the staircase. However, because they were so familiar with the pattern, they could also access it virtually, which would help with debugging and logical reasoning later. A role-play game to establish computational thinking was introduced before any actual programming began. This allowed the children to familiarise themselves with the function calls successor and predecessor (succ, pred) while traversing the staircase. The children would take turns to ask each other questions such as succ White (answer Red) or pred Orange (answer Blue). To help the children access the new programming vocabulary, one child would take the role of the ‘engine’ and the other ‘interpreter’. This was extended to succ $ succ (successor of successor) which uses function composition $. Some children could creatively enjoy inventing functions such as succ $ succ $ succ White to test the technique. The children experimented by writing down their calls before building themselves a rod calculator in Haskell. The language soon became a natural part of lessons and meant that it was easy to progress to the next step.</h2>

```

Comment:
`h2` ?

Reply:
why "successor" and other words with different fonts where not placed in inline elements?
![image](https://user-images.githubusercontent.com/104353194/165760453-d623be4b-6608-4d44-ac51-2da9922ddc42.png)

Reply:
Please look through all the text and add it for all places where text differs from the main font